### PR TITLE
format changes and code clean-up

### PR DESCRIPTION
**Pull request by saipj**
This PR address the following:
1.  Readme changes for default OpenVINOProviderOptions
2.  Format changes
3.  code clean-up

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -114,8 +114,8 @@ Once you have created your environment, using Python, execute the following step
 ## Add ONNX Runtime for PyTorch to your PyTorch inference script
 
 ```python
-from torch_ort import ORTInferenceModule
-model = ORTInferenceModule(model)
+from torch_ort import ORTInferenceModule,OpenVINOProviderOptions
```
Comment (ynimmaga) [inline Readme.md]: Suggest adding a space before "OpenVINOProviderOptions"

File: torch_ort_inference/torch_ort/ortinferencemodule/ortinferencemodule.py [hunk 2]
```diff
@@ -13,11 +13,10 @@
 
 import onnxruntime
 from onnxruntime.capi import _pybind_state as C
-from onnxruntime.training import ortmodule
-from onnxruntime.training.ortmodule import _io, _logger, _onnx_models, _utils
+from onnxruntime.training.ortmodule import _io, _onnx_models, _utils
 from onnxruntime.training.ortmodule.debug_options import DebugOptions, LogLevel
 
-from .provider_options import ProviderOptions,OpenVINOProviderOptions
```
Comment (ynimmaga) [inline torch_ort_inference/torch_ort/ortinferencemodule/ortinferencemodule.py:20]: Suggest adding a space before "OpenVINOProviderOptions"

File: Readme.md (hunk 1 above)
Comment (saipj) [reply Readme.md]: Not applicable anymore with latest changes.

File: torch_ort_inference/torch_ort/ortinferencemodule/ortinferencemodule.py (hunk 2 above)
Comment (saipj) [reply torch_ort_inference/torch_ort/ortinferencemodule/ortinferencemodule.py:20]: Not applicable anymore with latest changes.

**Review by ynimmaga — APPROVED** [review]
LGTM

**Pull request merged**